src/routes/admin/_admin.webhooks.tsx

```diff
@@ -0,0 +1,50 @@
+import {
+  CustomErrorComponent,
+  Tabs,
+  TabsContent,
+  TabsList,
+  TabsTrigger,
+  Toaster,
+  WebhooksTabContent,
+} from '@/components';
+import { useSpvWalletClient } from '@/contexts';
+
+import { addStatusField, webhooksQueryOptions } from '@/utils';
+import { useSuspenseQuery } from '@tanstack/react-query';
+import { createFileRoute } from '@tanstack/react-router';
+import { useState } from 'react';
+
+export const Route = createFileRoute('/admin/_admin/webhooks')({
+  component: Webhooks,
+  errorComponent: ({ error }) => <CustomErrorComponent error={error} />,
+});
+
+export function Webhooks() {
+  const { spvWalletClient } = useSpvWalletClient();
+  const [tab, setTab] = useState<string>('all');
```

Comment:
I think here we don't need to type the string. Ts will infer the type. Please correct me if I'm wrong

Reply:
Yes. Removed string type